OWML.Common/IModInputCombination.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using UnityEngine;
+
+namespace OWML.Common
+{
+    public interface IModInputCombination
+    {
+        float LastPressedMoment { get; }
+        float PressDuration { get; }
+        string ModName { get; }
+        string Name { get; }
+        ReadOnlyCollection<KeyCode> Singles { get; }
+        ReadOnlyCollection<long> Hashes { get; }
+        bool IsFirst { get; }
+        [Obsolete("For internal use")]
```

Comment:
I get what you're trying to achieve, but it's not Obsolete :P 
I think the best approach would be to separate it from the above properties with a newline, then explain what to use and not in the wiki.

Reply:
I renamed it to InternalIsFirst and removed the attribute